source/ReactLoadableSSRAddon.js

```diff
@@ -226,12 +226,14 @@ export default class ReactLoadableSSRAddon {
             );
           }
 
-          assets[id][ext].push({
-            file,
-            hash,
-            publicPath: url.resolve(this.options.publicPath || '', file),
-            integrity: currentAsset[this.options.integrityPropertyName],
-          });
+          if (currentAsset) {
```

Comment:
**You got the right direction!** 

But we can avoid adding an extra `if statement` by adding a `short-circuit evaluation` in the `variable declaration`.

```js
const currentAsset = originAssets[file] || {};
```

Could you please, `update` & `test` your branch against the above condition?

Reply:
Was going to point this out too with `inline if/else condition` but the author of this package respond extremely fast !

Reply:
@endiliey 😬

We got keep things in motion hahaha

Reply:
@nguyenngocphuongnb

Reply:
@themgoncalves your suggestion work fine, i have updated the PR, thank you for pointing this out

Reply:
@nguyenngocphuongnb nice!